src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/CreateCohortController.cs

```diff
@@ -131,6 +143,7 @@ public async Task<IActionResult> ConfirmProvider(ConfirmProviderRequest request)
 
         [Route("confirm-provider")]
         [HttpPost]
+        [ValidateAntiForgeryToken]
         public IActionResult ConfirmProvider(ConfirmProviderViewModel request)
         {
             var validationResult = _confirmProviderViewModelValidator.Validate(request);
```

Comment:
This explicit call can be removed

Reply:
Done, removed all and added global filter